ilastik/workflows/neuralNetwork/_localWorkflow.py

```diff
@@ -0,0 +1,248 @@
+import argparse
+import logging
+
+import numpy
+
+from ilastik.config import cfg
+from ilastik.workflow import Workflow
+from ilastik.applets.dataSelection import DataSelectionApplet
+from ilastik.applets.serverConfiguration.types import ServerConfig, Device
+from ilastik.applets.neuralNetwork import NNClassApplet, NNClassificationDataExportApplet
+from ilastik.applets.batchProcessing import BatchProcessingApplet
+from ._localLauncher import LocalServerLauncher
+
+from lazyflow.operators import tiktorch
+
+from lazyflow.graph import Graph
+
+
+logger = logging.getLogger(__name__)
+
+
+class LocalWorkflow(Workflow):
+    """
+    This class provides workflow for a remote tiktorch server
+    It has special server configuration applets allowing user to
+    connect to remotely running tiktorch server managed by user
+    """
+
+    auto_register = False
```

Comment:
why is it disabled?

Reply:
Because registration is conditional dependent on local tiktorch availability

Reply:
well, in any case, it doesn't work currently :) at least not for me if I supply tiktorch-server exec via cmd (verified that it's indeed picked up, didn't debug into it further).